src/constants.js

```diff
@@ -72,3 +73,8 @@ export const attributeDataType = {
 	entityReference: "EntityReference",
 	customType: "CustomType",
 };
+
+export const definitionType = {
```

Comment:
Use just string values, and when you render - based on values you will get localization strings